### PR TITLE
Fix update queue status failed.

### PR DESCRIPTION
**Pull request by ZejiaJiang**
# Description
Fixed https://github.com/Azure/azure-sdk-for-java/issues/30880
The updated xml should have exact same order as what we get from service. Move `MaxMessageSizeInKilobytes` to the end of xml.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by ZejiaJiang** [comment]
I wonder do I need update the session record folder? if needed, is there a command or I just update manually?

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/models/QueueDescription.java [hunk 1]
```diff
@@ -96,14 +96,6 @@ public final class QueueDescription {
...
 
-    /*
-     * The maximum message size for a message.
-     */
-    @JacksonXmlProperty(
-        localName = "MaxMessageSizeInKilobytes",
-        namespace = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect")
-    private Long maxMessageSizeInKilobytes;
-
     /*
```
Comment (liukun-msft) [inline sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/models/QueueDescription.java:99]: The `QueueDescription` is generated by swagger, here is the [README ](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/servicebus/azure-messaging-servicebus/swagger)for how to generate.

However, I find the swagger.json file is out of date and we done some manually changes on the generated models (`maxMessageSizeInKilobytes` is one of them).  I created #32149 to keep updating our swagger. Before that, I think you can try to update `MaxMessageSizeInKilobytes` field in swagger file, regenerate and ignore other changes.

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/models/QueueDescription.java [hunk 2]
```diff
@@ -96,14 +96,6 @@ public final class QueueDescription {
...
      * Value that indicates whether server-side batched operations are enabled.
```
Comment (liukun-msft) [inline sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/models/QueueDescription.java:100]: Does the `TopicDescription` has the same problem?

Comment (ZejiaJiang) [reply sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/models/QueueDescription.java:100]: Nope, I check the `MaxMessageSizeInKilobytes` PR  https://github.com/Azure/azure-sdk-for-java/pull/25147 , in `TopicDescription` this field is added to the end.

File: sdk/servicebus/azure-messaging-servicebus/swagger/servicebus-swagger.json [hunk 3]
```diff
@@ -767,6 +767,15 @@
             "name": "ForwardDeadLetteredMessagesTo",
             "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
           }
+        },
+        "MaxMessageSizeInKilobytes": {
```
Comment (ki1729) [inline sdk/servicebus/azure-messaging-servicebus/swagger/servicebus-swagger.json:771]: Thanks for making this change as well. We need to make sure that swagger is always in line with the SDK changes

File: sdk/servicebus/azure-messaging-servicebus/src/test/resources/session-records/getQueue.json [hunk 4]
```diff
@@ -12,7 +12,7 @@
...
...eDetectionHistoryTimeWindow><MaxDeliveryCount>20</MaxDeliveryCount><EnableBatchedOperations>true</EnableBatchedOperations><SizeInBytes>11985584</SizeInBytes><MessageCount>59942</MessageCount><IsAnonymousAccessible>false</IsAnonymousAccessible><AuthorizationRules></AuthorizationRules><Status>Active</Status><CreatedAt>2022-11-09T03:17:33.977Z</CreatedAt><UpdatedAt>2022-11-15T08:48:26.613Z</UpdatedAt><AccessedAt>2022-11-10T07:09:16.985Z</AccessedAt><SupportOrdering>true</SupportOrdering><CountDetails xmlns:d2p1=\"http://schemas.microsoft.com/netservices/2011/06/servicebus\"><d2p1:ActiveMessageCount>59942</d2p1:ActiveMessageCount><d2p1:DeadLetterMessageCount>0</d2p1:DeadLetterMessageCount><d2p1:ScheduledMessageCount>0</d2p1:ScheduledMessageCount><d2p1:TransferMessageCount>0</d2p1:TransferMessageCount><d2p1:TransferDeadLetterMessageCount>0</d2p1:TransferDeadLetterMessageCount></CountDetails><AutoDeleteOnIdle>P10675199DT2H48M5.477S</AutoDeleteOnIdle><EnablePartitioning>false</EnablePartitioning><EntityAvailabilityStatus>Available</EntityAvailabilityStatus><EnableExpress>false</EnableExpress><MaxMessageSizeInKilobytes>1024</MaxMessageSizeInKilobytes></QueueDescription></content></entry>",
```
Comment (ki1729) [inline sdk/servicebus/azure-messaging-servicebus/src/test/resources/session-records/getQueue.json:15]: There might be changes to other session recordings as well, please make sure these test are functional

**Review by ki1729 — APPROVED** [review]
Looks good overall in terms of the change. Please make sure the tests are good to go as well

File: sdk/servicebus/azure-messaging-servicebus/swagger/servicebus-swagger.json (hunk 3 above)
Comment (ZejiaJiang) [reply sdk/servicebus/azure-messaging-servicebus/swagger/servicebus-swagger.json:771]: Yeah, I run autorest.java to generate sdk by using these swagger, everything is ok.

File: sdk/servicebus/azure-messaging-servicebus/src/test/resources/session-records/getQueue.json (hunk 4 above)
Comment (ZejiaJiang) [reply sdk/servicebus/azure-messaging-servicebus/src/test/resources/session-records/getQueue.json:15]: all tests are passed in CI and my local environment. Are there other things that I need to do?

**Pull request merged**